horovod/ray/elastic_v2.py

```diff
@@ -162,6 +162,11 @@ class ElasticParams(BaseParams):
         reset_limit (int): Maximum number of times that the training
             job can scale up or down the number of workers after
             which the job is terminated.
+        cooldown_range(Tuple[int, int]): Range of seconds(min, max) a failing
+            host will remain in blacklist.
+            e.g: cooldown_ranage: [10, 100]. This sets the minimum cooldown
```

Comment:
Well, not that this is a tuple, the example should read `(10, 100)`. Also the notation `cooldown_ranage:` is not really useful as this is not how you would call this method with the example value.

```suggestion
            Example: cooldown_ranage=(10, 100)
            This sets the minimum cooldown
```

Reply:
Done